src/app/helptext/directoryservice/ldap.ts

```diff
@@ -63,16 +63,25 @@ ldap_kerberos_principal_tooltip: T('Select the location of the principal in the
 
 ldap_ssl_name : 'ssl',
 ldap_ssl_placeholder : T('Encryption Mode'),
-ldap_ssl_tooltip: T('Authentication only functions when a Certificate\
- is selected with either the <i>SSL</i> or <i>TLS</i> option.'),
+ldap_ssl_tooltip: T('Options for encrypting the LDAP connection: \
+ <ul> \
+ <li><i>OFF:</i> Do not encrypt the LDAP connection.</li> \
+ <li><i>ON:</i> <i>SSL</i> encrypts the LDAP connection and uses port \
+ <i>636</i>.</li> \
+ <li><i>START_TLS:</i> <i>STARTTLS</i> encrypts the LDAP connection and \
+ uses the default LDAP port <i>389</i>.</li> \
+ </ul>'),
 
 ldap_certificate_name : 'certificate',
 ldap_certificate_placeholder : T('Certificate'),
-ldap_certificate_tooltip: T('Select the LDAP CA certificate. The certificate for the\
- LDAP server CA must first be imported using the System/Certificates menu.'),
+ldap_certificate_tooltip: T('Certificate to use when performing LDAP \
+ certificate-based authentication. To configure LDAP certificate-based \
+ authentication, create a Certificate Signing Request for the LDAP \
+ provider to sign. A certificate is not required when using \
+ username/password or Kerberos authentication.'),
 
 ldap_validate_certificates_placeholder: T('Validate Certificates'),
-ldap_validate_certificates_tooltip: T('Check server certificates in a TLS session.'),
+ldap_validate_certificates_tooltip: T('Checks certificate authenticity.'),
```

Comment:
Present tense and better verb:
```suggestion
ldap_validate_certificates_tooltip: T('Verify certificate authenticity.'),
```